### PR TITLE
yaml: virtio: AGL: wrap domain-specific variables under switches

### PR DESCRIPTION
**Pull request by rshym**
This commit adds files missed by
b7b0622561195d0f6012c3668fd8a7a5ecc28205.

Fixes:
b7b0622561195d0f6012c3668fd8a7a5ecc28205.

### CODE REVIEW

File: prod-devel-rcar-virtio.yaml [hunk 1]
```diff
@@ -614,6 +613,9 @@ parameters:
         components:
           dom0:
             builder:
+              conf:
+                - [XT_DOMU_DTB_NAME, "%{XT_DOMU_DTB_NAME}"]
```
Comment (otyshchenko1) [inline prod-devel-rcar-virtio.yaml:617]: Why this is needed? There is no dtb for DomU at all AFAIR

**Review by lorc — COMMENTED** [review]
genuine question: is AGL build viable at all? I.e. do we need it? Was it tested in last year? 

Also, it would be nice to extend commit description, maybe including parts from  b7b0622561195d0f6012c3668fd8a7a5ecc28205, because current description just does not provide enough information by itself.

Also-also, there is quite specific format for the `Fixes:` tag. It is described in this section: https://www.kernel.org/doc/html/v4.17/process/submitting-patches.html#describe-your-changes

**Comment by rshym** [comment]
Not needed 'XT_DOMU_DTB_NAME' removed from the virtio.yaml.
The commit message was reworked.
PR was rebased on the latest master.

File: prod-devel-rcar-virtio.yaml (hunk 1 above)
Comment (rshym) [reply prod-devel-rcar-virtio.yaml]: Thanks, you are right. This is my copy-paste error.

**Comment by rshym** [comment]
> genuine question: is AGL build viable at all? I.e. do we need it? Was it tested in last year?

IMO, prod-devel-rcar with AGL is not compilable right due to the following reasons:
- AGL sources are dunfell-based;
- android-related part of the image needs to be aligned with regular prod-devel.yaml.

To fix this we need to update AGL to a Kirkstone based release and update Android-related part.

So, before the start of all the latest updates to agl.yaml, I decided, that I have two options:
- do not touch agl.yaml, and pull all updates only when we will update the AGL part of the yaml;
- apply all changes together with all other yamls, despite AGL-related part is not compilable.

I decided to apply changes now,when I understand what I'm doing.

**Review by lorc — COMMENTED** [review]
Okay, fair enough.

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
`Ack` is added.
Smoke-tested.

**Pull request merged**